doc/setup_instructions.md

```diff
@@ -579,6 +579,24 @@ is available through Homebrew.
 `msgmerge` is part of the gettext tools. So any errors with that
 missing are about these gettext tools.
 
+When you add new translation strings to the game, you need to run 
```

Comment:
I think this is duplicating slightly too much information from `working_with_translations.md`. So I think this explanation here should be shortened a bit to be just a tl;dr; from that other file. And the primary information should be put into that file so that all translations info is contained in one place. Though I do think that mentioning how to install the merge driver would be good here as this setup instructions file does deal with git options.

Reply:
I have moved some text there